test/regex-filter.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>QUnit Regex Filtering Test Suite</title>
+    <link rel="stylesheet" href="../dist/qunit.css">
+    <script src="../dist/qunit.js"></script>
+    <script>
+    QUnit.config.filter = '/foo|bar/i'
+    </script>
+    <script src="regex-filter.js"></script>
+  </head>
+  <body>
+    <div id="qunit"></div>
+  </body>
+</html>
```

Comment:
Would you change the indentation from space to tabs? The same goes to:

- test/regex-filter.js
- test/string-filter.html
- test/string-filter.js